lib/vintage_net_mobile/modem/huawei_E3372.ex

```diff
@@ -0,0 +1,88 @@
+defmodule VintageNetMobile.Modem.HuaweiE3372 do
+  @behaviour VintageNetMobile.Modem
+
+  alias VintageNet.Interface.RawConfig
+  alias VintageNetMobile.{CellMonitor, Chatscript, ExChat, ModemInfo, PPPDConfig, SignalMonitor}
+  alias VintageNetMobile.Modem.Utils
+
+
+  @moduledoc """
+  Huawei E3372 support
+
+  This modem needs a usb modeswitch, the follwing can serve as an example
+
+  ```elixir
+  def modemswicth(product_id \\ "14fe") do
+      Toolshed.cmd("usb_modeswitch -v 12d1 -p #{product_id} -X")
```

Comment:
Also you should escape the interpolation: `\#{…}`. Documentation is just normal strings.